### PR TITLE
Toggle movie to favorite.

### PR DESCRIPTION
**Pull request by pink-bunny**

### CODE REVIEW

File: src/pages/Movie/TopInfo/ToggleToFavorite/__tests__/containet.test.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  const container = wrapper.dive();
+  const instance = container.instance();
+
+  it('matches snapshot', () => {
+    expect(container).toMatchSnapshot();
+  });
+
+  it('changeFavoriteStatus() trigger toggleToFavorite()', () => {
+    const {
+      id,
```
Comment (plzen) [inline src/pages/Movie/TopInfo/ToggleToFavorite/__tests__/containet.test.js]: You already have this in `props` variable

File: src/pages/Movie/TopInfo/ToggleToFavorite/__tests__/containet.test.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import React from 'react';
```
Comment (plzen) [inline src/pages/Movie/TopInfo/ToggleToFavorite/__tests__/containet.test.js]: Please, rename this file from `containet.test.js` to `container.test.js`

File: src/pages/Movie/TopInfo/ToggleToFavorite/__tests__/containet.test.js (hunk 1 above)
Comment (pink-bunny) [reply src/pages/Movie/TopInfo/ToggleToFavorite/__tests__/containet.test.js]: Fixed

File: src/pages/Movie/TopInfo/ToggleToFavorite/__tests__/containet.test.js (hunk 2 above)
Comment (pink-bunny) [reply src/pages/Movie/TopInfo/ToggleToFavorite/__tests__/containet.test.js]: Done

**Pull request merged**